reference/admin.yaml

```diff
@@ -589,7 +588,13 @@ components:
             string by default if null is passed.
           example: Enter vendor
         category_ids:
-          $ref: '#/components/schemas/category_ids'
+          allOf:
```

Comment:
Before (main branch):
<img width="665" alt="Screenshot 2022-05-13 at 2 27 27 PM" src="https://user-images.githubusercontent.com/22430409/168249031-bdf62e31-a5cd-4f90-884d-22d56732aa87.png">

Reply:
After this change (pvt branch) :
<img width="649" alt="Screenshot 2022-05-13 at 2 27 08 PM" src="https://user-images.githubusercontent.com/22430409/168249119-6fef3d4b-fc52-4b4e-99d6-d48d8d39c4b6.png">